.github/workflows/refapp-3x-visit.yml

```diff
@@ -0,0 +1,22 @@
+name: RefApp 3.x Clinical Visit
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  repository_dispatch:
+    types: [ qa ]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Using Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Installing dependencies
+        run: npm install
+      - name: Run clinical visit workflow tests
+        run: npm run refApp3Visit
```

Comment:
You can make the visit be more clear since we can have different types of visits in the module. 
So something like `npm run refapp3ClinicalVisit`

Reply:
Thanks for the suggestion, @kdaud 

fixed with 59d7790